src/app/pages/NFTInstanceDashboardPage/InstanceImageCard.tsx

```diff
@@ -91,11 +94,28 @@ export const InstanceImageCard: FC<InstanceImageCardProps> = ({ isFetched, isLoa
             display: 'flex',
             flexDirection: 'column',
             alignItems: 'center',
+            justifyContent: 'space-between',
+            minHeight: imageSize,
           }}
         >
           {isLoading && <Skeleton variant="rectangular" width={imageSize} height={imageSize} />}
-          {isFetched && nft && !isNftImageUrlValid(nft.image) && <NoPreview placeholderSize={imageSize} />}
-          {isFetched && nft && isNftImageUrlValid(nft.image) && (
+          {isFetched && nft && imageLoadError && (
+            <Box sx={{ flex: 1, display: 'flex', alignItems: 'center' }}>
+              <Box
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  alignItems: 'center',
+                }}
+              >
+                <NoPreview placeholderSize={imageSize} />
+                <Link href={nft.image} rel="noopener noreferrer" target="_blank">
```

Comment:
This looks vulnerable to `image="javascript:alert(1)"`

Reply:
Can our backend can send this instead of ipfs url? Anyway, you mean to show Link only when isNftImageUrlValid=true, right?

Reply:
I'm not sure if backend allows it, but I would prefer to keep validProtocols on frontend too

Reply:
Luckily react says `Warning: A future version of React will block javascript: URLs as a security precaution.` otherwise I might add
```tsx
const originalSetAttribute = Element.prototype.setAttribute
Element.prototype.setAttribute = function (...args) {
  const [name, value] = args
  if ((name === 'href' || name === 'src') && !isNftImageUrlValid(value)) {
    console.error('blocked suspicious setAttribute', src, value)
    return
  }
  return originalSetAttribute.apply(this, args)
}
```

Reply:
related issues in wallet from validator metadata:
https://github.com/oasisprotocol/oasis-wallet-web/pull/199
https://github.com/oasisprotocol/oasis-wallet-web/issues/185

Reply:
Ok for now I will merge as is, because there is `isNftImageUrlValid` condition that prevents rendering a link.